ta4j-core/src/main/java/org/ta4j/core/BacktestingResult.java

```diff
@@ -0,0 +1,93 @@
+package org.ta4j.core;
+
+import org.ta4j.core.analysis.criteria.*;
+import org.ta4j.core.num.Num;
+
+/**
+ * This class contains the result of a backtested strategy
+ */
+public class BacktestingResult implements Comparable<BacktestingResult> {
```

Comment:
As a class name says it should be a result, so it means this class should be a simple dto without business logic, it should be placed in other class

Reply:
skip